model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/PersistentUserSessionProvider.java

```diff
@@ -884,27 +876,14 @@ private UserSessionEntity createUserSessionEntityInstance(UserSessionModel userS
 
 
     private AuthenticatedClientSessionAdapter importClientSession(UserSessionAdapter sessionToImportInto,
```

Comment:
If we remove the "offline" boolean flag, the name of the method should change to "importOfflineClientSession" to avoid people using this method in a wrong place.

Reply:
Makes sense. Updated!